### PR TITLE
📝 readme tweaks

### PR DESCRIPTION
**Pull request by filmaj**
## Description
- formatting consistency
- added TOC
- moved "where is JS" to top
- made notes when sections contained adobe-only firewall links
- linked to contribution guidelines, removed 'learn more' section that linked to internal top-level spectrum docs

## Related Issue
This issue relates to #17, but I am not sure that it closes it.

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,33 @@
...
+The CSS source files also contain [Topdoc](https://github.com/Topdoc/topdoc) comments with a placeholder for documentation values that are injected at build time. The source of those injected values is found in the YAML formatted files in the `docs` folder. A key part of the docs data is the `markup` node, which contains the HTML elements needed to apply the corresponding element selectors and render the elements as generated Topdoc output.
 
-A successful build will create a `dist` folder.  The `dist/docs` folder is where the Topdoc output and related template files will end up.
+A successful build will create a `dist` folder. The `dist/docs` folder is where the Topdoc output and related template files will end up.
 
 ### Updating internal dependencies
 
+> Internal-to-Adobe only - for now
```
Comment (GarthDB) [inline README.md]:
```suggestion
*Internal-to-Adobe only - for now*
```
Just slightly more semantic to use emphasis vs a block quote syntax.

File: README.md [hunk 2]
```diff
@@ -144,6 +158,8 @@ Finally, you'll need workflow icons for your app. The `icons/spectrum-icons.svg`
 AdobeSpectrum.loadIcons('icons/spectrum-icons.svg');
 ```
 
+> NOTE: the following link is internal-to-Adobe-only
```
Comment (GarthDB) [inline README.md]:
```suggestion
***NOTE**: the following link is internal-to-Adobe-only*
```

File: README.md [hunk 3]
```diff
@@ -246,6 +267,8 @@ As long as the component name matches one of the folder names within `dist/compo
 
 ### Remote Testing with the Test Server
 
+> Internal-to-Adobe only - for now
```
Comment (GarthDB) [inline README.md]:
```suggestion
*Internal-to-Adobe only - for now. Follow progress on [this issue](https://github.com/adobe/spectrum-css/issues/23).*
```

**Comment by filmaj** [comment]
Thanks for the suggestions, I've accepted them!

**Pull request merged**